draft-ietf-quic-transport.md

```diff
@@ -3157,30 +3158,35 @@ after a connection is established and 1-RTT keys are available; see
 
 ## Protected Packets {#packet-protected}
 
-All QUIC packets except Version Negotiation packets use authenticated encryption
-with associated data (AEAD) {{!RFC5116}} to provide confidentiality and
-integrity protection.  Retry packets use AEAD to provide integrity protection.
-Details of packet protection are found in {{QUIC-TLS}}; this section includes an
-overview of the process.
-
-Initial packets are protected using keys that are statically derived. This
-packet protection is not effective confidentiality protection.  Initial
-protection only exists to ensure that the sender of the packet is on the network
-path. Any entity that receives the Initial packet from a client can recover the
-keys necessary to remove packet protection or to generate packets that will be
+QUIC packets have different levels of cryptographic protection based on the
+type of packet. Details of packet protection are found in {{QUIC-TLS}}; this
+section includes an overview of the protections that are provided.
+
+Version Negotiation packets have no cryptographic protection; see
+{{QUIC-INVARIANTS}}.
+
+Retry packets use an authenticated encryption with associated data (AEAD;
+{{?AEAD=RFC5116}}) function to protect against accidental modification.
+
+Initial packets use an AEAD with keys that are statically derived. This does
```

Comment:
The keys are generated for each connection, so I think it is confusing to use the word "statically" here. Maybe something like:

```suggestion
Initial packets use an AEAD with keys that are derived using a value that is visible on the wire, and therefore does
```